mod.ts

```diff
@@ -99,8 +99,13 @@ export class App {
         Object.assign(params, parseURLSearchParams(search));
       }
     } else {
-      const decodedBody = decode(await readAll(req.body)); // FIXME: this line is should be refactored using Deno.Reader
-      const contentType = req.headers.get('content-type');
+      const rawContentType = req.headers.get('content-type') || "application/octet-stream";
+      const [contentType, ...typeParamsArray] = rawContentType.split(';')
+      const typeParams = Object.assign({}, ...typeParamsArray.map(params => params.split('=')))
```

Comment:
This code looks not correct, how about using `reduce`?
```suggestion
      const typeParams = typeParamsArray.reduce((params, curr) => {
        const [key, value] = curr.split('=');
        params[key] = value;
        return params;
      }, {});
```
Please see this: https://jsbin.com/verabuh/edit?js,console

Reply:
Indeed...! Tests I wrote are passed but the default value of `params['charset']` just worked... 😓